js/lms-ui-fileview.js

```diff
@@ -0,0 +1,78 @@
+﻿/*
+ * LMS version 1.11-git
+ *
+ *  (C) Copyright 2001-2022 LMS Developers
+ *
+ *  Please, see the doc/AUTHORS for more information about authors!
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License Version 2 as
+ *  published by the Free Software Foundation.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307,
+ *  USA.
+ *
+ *  $Id$
+ */
+
+function lmsFileView(file, view_containerid) {
+    if (typeof (file) == 'undefined') {
+        alert("404, Error: no file to view");
+        return;
+    }
+
+    var objUrl = (window.URL ? URL : webkitURL).createObjectURL(file);
```

Comment:
No tak - po co jQuery - lepiej po raz kolejny wynajdywać koło na nowo :)

Reply:
Skorzystałem ze wzoru na jednej ze stron. Jeśli masz pomysł jak to uprościć to poproszę  -chętnie skorzystam z podpowiedzi.

Reply:
Jak proponujesz?